src/index.html

```diff
@@ -3,11 +3,55 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <img
+        src="../src/images/imac.jpeg"
+        class="card__img"
+        alt="Monitir picture"
+      >
+
+      <section>
+        <h2 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__code">Product code: 195434</p>
+      </section>
```

Comment:
This is only 1 block, and the section tag is usually used for a whole section with content, so it doesn't need to be used here
```suggestion
      <h2 class="card__title"> APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
      <p class="card__code">Product code: 195434</p>
```